### DISCUSSION TITLE
RangeCalendar no longer supports a null value?

### DISCUSSION THREAD

**Discussion by Stanaland-Michael-bah** [adobe/react-spectrum#6776] [General]
Question probably for @snowystinger ... 

In general, the date pickers and calendars accept values that are either a `DateValue` or `null`, but it appear's the `null` option was lost for RangeCalendar.

Looks like it was this [gnarly, not-at-all-fun PR](https://github.com/adobe/react-spectrum/pull/6076/files) ... the RangeCalendarProps interface lost the `ValueBase<RangeValue<T> | null>` bit. 

Just wanted to confirm this was an intentional change, as it surprised me when I updated react-aria today.

**Comment by snowystinger** [comment]
Looks like that was accidental, as a result of a gnarly, not-at-all-fun change as you pointed out, haha. Thank you for reporting it. I've opened a PR to fix it. https://github.com/adobe/react-spectrum/pull/6790 and addressed a few more things that fell out of testing this.
